pkg/odo/cli/service/create.go

```diff
@@ -109,7 +112,15 @@ func (o *ServiceCreateOptions) Complete(name string, cmd *cobra.Command, args []
 		o.interactive = true
 	}
 
-	if o.componentContext != "" {
+	if experimental.IsExperimentalModeEnabled() {
+		envInfo, err := envinfo.NewEnvSpecificInfo(o.componentContext)
+		if err != nil {
+			return errors.Wrap(err, "unable to retrieve configuration information")
+		}
+
+		o.EnvSpecificInfo = envInfo
+		o.Context = genericclioptions.NewDevfileContext(cmd)
```

Comment:
Since occlient isn't there, it panics when interactive mode is triggered https://github.com/openshift/odo/pull/3509#issuecomment-662395711

Reply:
I think we can also have an option to provide the name interactively.

Reply:
Interactive mode is going to be a completely different task. We're not looking at it for v2. Mainly because there's not completely clarity about the approach we should take for it.

Reply:
Ok 👍